packages/blade/src/components/Typography/Text/getTextStyles.ts

```diff
@@ -11,8 +11,9 @@ const getTextStyles = <T extends { variant: TextVariant }>({
   size,
   contrast,
   theme,
-}: Pick<TextProps<T>, 'type' | 'variant' | 'weight' | 'size' | 'contrast'> & {
+}: Pick<TextProps<T>, 'type' | 'variant' | 'size' | 'contrast'> & {
```

Comment:
why remove from here? weight is there na on textprops?

Reply:
Yes removed from here because in the TextProps the weight is `bold | regular` but getBaseTextStyles's types accept `bold | regular | semibold` since we added `semibold` in the global font weight token.